frontend/src/routes/home/Home.tsx

```diff
@@ -52,47 +55,53 @@ function Home(props) {
     return nextDays
   }
 
-  return content.loading ? (
-    <div className="spinner-border" role="status" />
-  ) : (
-    <div className="home-page">
-      <div className="fullScreen-wrapper">
-        <div className="left-title-section">
-          <span className="bold-title">{content.title}</span>
-          <div className="parag">{content.description}</div>
-          <div className="default-red-button">
-            <Link to="/program" onClick={() => setActiveRoute('program')}>
-              AFLĂ PROGRAMUL
-            </Link>
+  const returnContent = () => {
+    return (
+      <>
+        <div className="fullScreen-wrapper">
```

Comment:
just a suggestion for the future, never mix camel-case with dashes in className. As a convention, classNames should always be dash separated. In this example you could choose `fullscreen-wrapper` or `full-scree-wrapper`.